fast/addons/2-networking-swp/outputs.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  tfvars = {
+    swp_cas_pool_ids = {
+      for k, v in module.cas : k => v.ca_pool_id
+    }
+    swp_gateway_ids = {
+      for k, v in module.swp : k => v.id
+    }
+    swp_gateway_security_policy_ids = {
+      for k, v in module.swp : k => v.gateway_security_policy
+    }
+    swp_service_attachment_ids = {
+      for k, v in module.swp : k => v.service_attachment
+    }
+  }
+}
+
+output "cas_pool_ids" {
+  description = "Certificate Authority Service pool ids."
```

Comment:
All the outputs here are missing tfdoc tags

Reply:
we don't use tfdoc tags in outputs anymore AFAIK (checked security and pf and none of those have them), reopen if I misinterpreted